src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -164,14 +163,10 @@ const personalDetailsSelector = (personalDetails) =>
  */
 export default React.memo(
     compose(
-        withReportCommentDrafts({
-            propName: 'comment',
-            transformValue: (drafts, props) => {
-                const draftKey = `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${props.reportID}`;
-                return lodashGet(drafts, draftKey, '');
-            },
-        }),
         withOnyx({
+            comment: {
+                key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${reportID}`,
+            },
```

Comment:
@adhorodyski this is already on main branch, and I think you can use that for part that Rory has mentioned https://github.com/Expensify/App/pull/30242#discussion_r1376599475